src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -853,8 +905,31 @@ int fx_muxer_t::read_config_and_execute(
         append_realpath(path, &probe_realpaths);
     }
 
+    // 'No candidate fx roll forward' is disabled by default. It can be enabled through:
+    // 1. Command line argument (--no-candidate-fx-roll-forward)
+    // 2. Runtimeconfig json file ('noCandidateFxRollForward' property)
+    // 3. DOTNET_NO_CANDIDATE_FX_ROLL_FORWARD env var
+    // The conflicts will be resolved by following the priority rank described above (from 1 to 3)
+    bool no_candidate_fx_roll_fwd_enabled = config.get_no_candidate_fx_roll_fwd();
```

Comment:
Please add a comment indicating that env var is processed in config file processing.